persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/PersistentEntityActor.scala

```diff
@@ -201,6 +201,12 @@ private[lagom] class PersistentEntityActor(
 
     case PersistentEntityActor.Stop =>
       context.stop(self)
+
+    /* The actor will receive a SaveSnapshotSuccess message when a snapshot
+     * was successfully saved. There's no need to do anything, but if we don't
+     * handle it, it will pollute the event stream with UnhandledMessage
+     * notifications */
```

Comment:
Really sorry to get stuck on this detail. I find the formatting still weird. Either we stick to javadoc formatting:

```
 /**
   * Stop the actor for passivation. `PoisonPill` does not work well
   * with persistent actors.
   */
```

(closing star-slash on a newline)

Or we use code comment formatting:

```
            // apply the event before persist so that validation exception is handled before persisting
            // the invalid event, in case such validation is implemented in the event handler.
```

Double-slash